Automattic/jetpack
# Media Endpoint: Do not auto-delete ZIP files

Originally, ZIP files were only allowed via the media endpoint as part of install flows, so Jetpack had a backstop to automatically delete ZIP files added via the API after a given amount of time.

WordPress.com/API now allows ZIP files (when permitted by the site), so the backstop is now deleting regular uploads.

Originally reported by @jmdodd in p1630077923064100-slack-CBG1CP4EN

#### Changes proposed in this Pull Request:

* Remove auto-deletion of ZIP uploads.

#### Jetpack product discussion


p9dueE-3ln-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Before and after patch, on a connected Jetpack site, upload a ZIP file via Calypso.
* Before patch: View the cron events after uploading and see the upgrader_scheduled_cleanup job scheduled.
* After patch: No cron event to cleanup.
⬆️ Resolves the bug. ⬇️ Ensures we don't create a new one.
* On a site that does not allow ZIP uploads (say, a subsite on a multisite where the network admin has set it to not accept zips), make sure you can't upload a ZIP.
* What are other ways to limit file type?
